slides.qmd

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Best Practices for Data Visualisation"
 author: "Andreas Krause, Nicola Rennie, and Brian Tarran"
-subtitle: "RSS International Conference 2023"
+subtitle: "RSS Annual Conference, Harrogate UK 2023"
```

Comment:
The official conference title is "RSS International Conference 2023" not "Annual" - see https://rss.org.uk/training-events/conference-2023/. Happy to add Harrogate after that.